src/fontra/views/editor/editor.js

```diff
@@ -910,6 +934,16 @@ export class EditorController {
     await writeToClipboard(clipboardObject);
   }
 
+  _copyToEventClipboard(event) {
+    if (!event) return;
```

Comment:
I don't see how this method gets called with a falsy event. Perhaps this line is redundant?